pysabnzbd/__init__.py

```diff
@@ -11,7 +11,7 @@ def __init__(self, base_url, api_key, web_root=None,
         if web_root is not None:
             web_root = '{}/'.format(web_root.strip('/'))
         else:
-            web_root = ''
+            web_root = 'sabnzbd'
```

Comment:
Did you test this change? Just looking at it, this will create a url like `http://host/sabnzbdapi`.

Reply:
Yes, locally, however I notice now the version on pypi is not the same as the latest version on github (one commit behind) and this pull request is meaningless. Sorry for taking up your time, it was a long day.

I will download the latest version from git instead.

Reply:
Sounds good. I will get the new version on PyPi today so it can be updated in home assistant too